swift/ql/lib/codeql/swift/frameworks/StandardLibrary/UrlSession.qll

```diff
@@ -0,0 +1,8 @@
+private import codeql.swift.dataflow.ExternalFlow
+
+private class UrlSessionSummaries extends SummaryModelCsv {
+  override predicate row(string row) {
+    row =
+      ";URLSession;true;dataTask(with:completionHandler:);;;Argument[0];Argument[1].Parameter[0];taint"
```

Comment:
Good point. Indeed, I agree that this should be a taint source as well. Having a taint summary will still be useful though (so that we can still track flow through these library calls in queries that don't pick any `RemoteFlowSource` as a taint source).

Reply:
I've added it to the taint source TODO list.